### PR TITLE
update Chinese file path as en_US

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -1 +1 @@
-*/_build
\ No newline at end of file
+**/_build
```
Comment (leckie-chn) [inline docs/.gitignore]: what's the point of replacing * with ** here?

Comment (squirrelsc) [reply docs/.gitignore]: ** means any folder depth, * means one level.

**Comment by squirrelsc** [comment]
They are mapping with en_us version, when en_us version is removed, they can be removed together.

**Comment by leckie-chn** [comment]
> They are mapping with en_us version, when en_us version is removed, they can be removed together.

@squirrelsc could you please just delete the mapping? After all, some of the documents is for developers, such as `deployment/docker/README.md`, which do not need a Chinese version.

**Pull request merged**